### PR TITLE
Updating README.md with text changes

### PR DESCRIPTION
**Pull request by jcasman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ![screenshot](docs/layout.png)
 
-This application does not use state management, but uses the [http](https://pub.dev/packages/http) package from dart to connect to the camera. Using the http commands, such as get and post, the application receives information and executes commands. Below is a list of this project's button controls using the [THETA API](https://api.ricoh/docs/theta-web-api-v2.1/). 
+This application does not use state management, but uses the [http](https://pub.dev/packages/http) package from Dart to connect to a RICOH THETA X camera. Using the http commands, such as GET and POST, the application receives information and executes commands. Below is a list of this project's button controls using the [THETA WebAPI](https://api.ricoh/docs/theta-web-api-v2.1/).
```
Comment (codetricity) [inline README.md:5]: nice catch that Dart is capitalized. https://dart.dev/guides
I also agree that GET and POST should be capitalized.  I guess we may as well capitalize HTTP as well.  :-)
https://developer.mozilla.org/en-US/docs/Web/HTTP

I don't know what the difference on GitHub is between writing a comment and starting a review.  I'm going with starting a review.

File: README.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ This application does not use state management, but uses the [http](https://pub.
 
 ## HTTP Package
 
-In order to use the HTTP package, I first needed to download it and import it at the top of my file. All HTTP requests require a url. For example, this is the url for getting the info from the camera. 
+In order to use the http package, I first needed to download it and import it at the top of my file. All HTTP requests require a url. For example, this is the url for getting the info from the camera.
```
Comment (codetricity) [inline README.md:15]: I agree that we should change it here.  As mentioned above, we should be consistent in the use of capitalization of HTTP.  We should also change it above.

**Review by codetricity — APPROVED** [review]
I also suggest changing the HTTP capitalization under the HTTP Package section

**Comment by jcasman** [comment]
> I also suggest changing the HTTP capitalization under the HTTP Package section

Agree, I've created another pull request to change it.

File: README.md (hunk 1 above)
Comment (caitlynbakery) [reply README.md:5]: Ok, I'll keep it in mind for the future.

File: README.md (hunk 2 above)
Comment (caitlynbakery) [reply README.md:15]: Alright, sounds good.

**Pull request merged**